modules/contracts/apps/challenge/examples/deploy.rs

```diff
@@ -0,0 +1,28 @@
+use cw_orch::{
+    anyhow,
+    prelude::{networks::parse_network, DaemonBuilder},
+    tokio::runtime::Runtime,
+};
+
+use abstract_challenge_app::{contract::CHALLENGE_APP_ID, ChallengeApp};
+use abstract_interface::AppDeployer;
+use semver::Version;
+
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+fn main() -> anyhow::Result<()> {
```

Comment:
Since this contract has the "abstract" namespace don't think we want to create account and claim namespace here. Unless for debugging that it deploys(and will have to change namespace before executing)